index.js

```diff
@@ -5,14 +5,21 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "Blue"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
-
+console.log('This is what the traveler object looks like: ', traveler)
+let img = document.querySelector(#profile.photo)
+let h2 = document.querySelector(#profile.name)
+let em = document.querySelector(#profile.nickname)
 
 /***** Deliverable 4 *****/
-
-
+let animalSightingList = 
+<li data-id="{animalSighting id}">
+  <p>{animal sighting description}</p>
+  <img src="{animal sighting photo}" alt="{animal sighting species}"/>
+  <a href="{animal sighting link}" target="_blank">Here's a video about the {animal sighting species} species!</a>
+</li>
 
 /***** Deliverable 5 *****/
```

Comment:
For deliverable 5 - the key here is first setting the data-id property on the `li` element when you're creating it in step 4. Remember, to set a data-* property on an HTML element using Javascript, we need to use the `dataset` property on the element. For example:
```Javascript
someElement.dataset.id = 4
```

Then here, you're going to want to search for that element using `querySelector`. Then once you find it, you can remove it by calling `remove()` on that element.